release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2839,3 +2839,30 @@ link:https://access.redhat.com/solutions/6955942[{product-title} 4.10.12 contain
 ==== Updating
 
 To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-10-13"]
+=== RHBA-2022:1690 - {product-title} 4.10.13 bug fix update
+
+Issued: 2022-05-11
+
+{product-title} release 4.10.13 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1690[RHBA-2022:1690] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1689[RHBA-2022:1689] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6956858[{product-title} 4.10.13 container image list]
+
+[id="ocp-4-10-13-known-issues"]
+==== Known issues
+* When installing {product-title} 4.10.13, the HPE nodes fail and are unable to start the inspection. Upgrade to {product-title} 4.9.28 or 4.10.9 to help mitigate the issue. Currently, there are plans to resolve this issue in a future version of {product-title}. For more information, see the following Knowledgebase solutions:
```

Comment:
Suggestion for singular form since there's just one solution listed here
```suggestion
* When installing {product-title} 4.10.13, the HPE nodes fail and are unable to start the inspection. Upgrade to {product-title} 4.9.28 or 4.10.9 to help mitigate the issue. Currently, there are plans to resolve this issue in a future version of {product-title}. For more information, see the following Knowledgebase solution:
```

Reply:
Thank you for catching that! PR has been updated. :)